CRBM-Viz/angular.json

```diff
@@ -63,6 +63,35 @@
                   "maximumError": "10kb"
                 }
               ]
+            },
+            "development": {
+              "fileReplacements": [
+                {
+                  "replace": "src/environments/environment.ts",
+                  "with": "src/environments/environment.dev.ts"
+                }
+              ],
+              "optimization": true,
```

Comment:
This might be very slow for development. Up to you, but we can change the optimization and AOT off since we dont be deploying using this environment